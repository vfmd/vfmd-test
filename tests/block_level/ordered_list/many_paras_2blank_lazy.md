1. One line
1. One line
Another line
And another


   Another para
Another line
And another
1. One line
