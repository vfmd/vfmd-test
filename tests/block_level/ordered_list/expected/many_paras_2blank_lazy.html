<ol>
<li>One line</li>
<li>One line
Another line
And another</li>
</ol>

<p>Another para
Another line
And another
 * One line</p>
